io/shared/src/main/scala/fs2/io/file/PathApi.scala

```diff
@@ -23,13 +23,46 @@ package fs2
 package io
 package file
 
+import cats.Show
+import cats.kernel.Hash
+import cats.kernel.Monoid
+import cats.kernel.Order
+
 private[file] trait PathApi {
-  def /(name: String): Path = resolve(name)
+
+  def /(name: String): Path
+
+  def /(path: Path): Path
+  
   def resolve(name: String): Path
+
+  def resolve(path: Path): Path
+  
   def normalize: Path
+
+  def isAbsolute: Boolean
+
+  def absolute: Path
+  
+  def names: Seq[Path]
```

Comment:
s/Seq/List/ so we have type class instances available

Reply:
I thought they got added for list?

Reply:
https://github.com/typelevel/cats/blob/main/core/src/main/scala/cats/instances/seq.scala

Reply:
Okay but still should be List - Seq offers no advantage here